cmd/app/options.go

```diff
@@ -45,6 +47,7 @@ func (o *options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Mask, "mask", "32", "mask part of the cidr")
 	fs.StringVar(&o.Kubeconfig, "kubeconfig", "", "kubeconfig to use with kubernetes client")
 	fs.StringVar(&o.Hostname, "hostname", "", "We will use os.Hostname if none provided")
+	fs.StringVar(&o.NodeFilter, "nodefilter", "fair", "We will use 'fair' if none provided")
```

Comment:
We can list them in help string and check manually. I've found examples where it is done. So, I suppose, it is not done automatically.